src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -3,6 +3,30 @@
 <html>
 	<head>
 		<script>
+
+			window.gitpodIDEErrors = []
+
+			const onVsCodeWorkbenchError = (event) => {
+				const flags = new Map();
+
+				// We take a look at what is the resource that was attempted to load;
+				const resourceSource = event?.target?.getAttribute('src') || event?.target?.getAttribute('href');
+
+				// If the event has a `target`, it means that it wasn't a script error
+				if (resourceSource) {
+					flags.set('resource', 'vscode-web-workbench');
+					flags.set('error', 'LoadError');
+				} else {
+					flags.set('error', 'Unknown');
+				}
+				window.gitpodIDEErrors.push(flags)
+			}
+
+			window.onerror = onVsCodeWorkbenchError;
```

Comment:
Is it necessary? it seems to make everything complicated? We will already assign it in supervisor-frontend. And there is no any script blocks before it to be executed.

Reply:
Removed this, I think you're right, since supervisor loads its script right away before others.